simple_ldap_user/simple_ldap_user.module

```diff
@@ -481,7 +481,16 @@ function simple_ldap_user_sync_user_to_drupal($drupal_user) {
   $ldap_user = SimpleLdapUser::singleton($drupal_user->name, TRUE);
 
   // Nothing to sync.
-  if (!$ldap_user->exists || (isset($drupal_user->uid) && $drupal_user->uid == 1)) {
+  if(isset($drupal_user->uid) && $drupal_user->uid == 1) {
+    // Check if user is an admin.
+    return;
+  } elseif (!$ldap_user->exists) {	//For other users.
+    $result = db_update('users')
+              ->fields(array(
+                  'status' => 0
+              )) //Update status to 0 for blocked users
+              ->condition('name',$drupal_user->name,'=')
+              ->execute();
```

Comment:
Now that we have blocked some users who do not exist in LDAP, do we also need an `elseif` here to unblock them if they subsequently reappear in LDAP?

Reply:
Yeah we need to add that condition.
So should I add elseif in which we update status to 1 if the user exists in ldap?

Reply:
Yes, I suspect if the user is disabled in Drupal, and if the user exists in LDAP, this is the right spot for changing the status back to enabled.  Is the Drupal user status just one or zero, or is it extensible by other modules?  If extensible, use the appropriate hooks here.

Reply:
User status is set only to 1 or 0.